src/Symfony/Component/Notifier/Bridge/Mailjet/composer.json

```diff
@@ -22,7 +22,7 @@
     "require": {
         "php": ">=7.2.5",
         "symfony/http-client": "^4.3|^5.0",
-        "symfony/notifier": "^5.3"
+        "symfony/notifier": "^5.4"
```

Comment:
because of the changes in `TransportTestCase` I need to bump symfony/notifier, or shall I send the `TransportTestCase` changes as bugfix to `5.3`. Note only Mailet bridge (introduced in `5.4` needs these changes).


@derrabus @nicolas-grekas

Reply:
This is fine imho.

Reply:
On second thought, the changes to the test case really look like a bugfix. So backporting them might be reasonable.

Reply:
Will do

Reply:
Here you go https://github.com/symfony/symfony/pull/41986

I will rebase this PR an revert `composer.json` after merge